_security/access-control/permissions.md

```diff
@@ -129,6 +129,50 @@ Users that have the permission [`restapi:admin/roles`]({{site.url}}{{site.baseur
 Keep in mind that enabling this feature and mapping system index permissions to normal users gives those users access to indexes that may contain sensitive information and configurations essential to a cluster's health. We also recommend caution when mapping users to `restapi:admin/roles` because this permission gives a user not only the ability to assign the system index permission to another user but also the ability to self-assign access to any system index.
 {: .warning }
 
+### `do_not_fail_on_forbidden`
+
+If the user attempts to query multiple indexes, some of which they do not have permission for, by default they will get an `error` in OpenSearch Dashboards UI or `exception` using `cURL` or API (failing to return the result for indexes which they **do** have access to). If you want the users to receive the hits from the search for only the indexes they have permission for, you can set the option `do_not_fail_on_forbidden` to `true` in `config.yml`. See the following example:
```

Comment:
```suggestion
If the user attempts to query multiple indices, some of which they lack permission for, by default they will get an `error` in OpenSearch Dashboards UI, or an `exception` using `cURL` or API. If instead, you want the user to receive the results from the search for any of the indices they _do_ have permission for, you can set the option `do_not_fail_on_forbidden` to `true` in `config.yml`. See the following example:
```

Reply:
@scrawfor99 - We actually DO use indexes, not indices, unless we are referring to the code itself. :)